webrtc.html

```diff
@@ -2363,11 +2363,14 @@
                 which point the <code><a>RTCPeerConnection</a></code> can fully
                 adopt the pending local description, or rollback to the current
                 description if the remote side rejected the change.</p>
+                <p>As noted in <span data-jsep="ice-candidate-pool">[[!JSEP]]</span>,
```

Comment:
From section 5.8:

>    o  If this m= section is new, begin gathering candidates for it, as
      defined in [RFC5245], Section 4.1.1, unless it has been marked as
      bundle-only.

>   o  Or, if the ICE ufrag and password values have changed, and it has
      not been marked as bundle-only, trigger the ICE agent to start an
      ICE restart, and begin gathering new candidates for the m= section
      as described in [RFC5245], Section 9.1.1.1.  If this description
      is an answer, also start checks on that media section as defined
      in [RFC5245], Section 9.3.1.1.